.ci/openvino-onnx/Jenkinsfile

```diff
@@ -4,6 +4,25 @@
 DOCKER_CONTAINER_NAME= "openvino-onnx-ci-container"
 DOCKER_IMAGE_TAG = "openvino-onnx-ci-image"
 
+// workaround for aborting previous builds on PR update
+@NonCPS
+def killPreviousRunningJobs() {
```

Comment:
Just a single correction is needed for better readability - `killPreviousRunningBuilds()` display the purpose better